packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/theme.js

```diff
@@ -1,20 +0,0 @@
-import { css } from 'styled-components'
-
-const theme = {
-  text: {
-    extend: props => css`
-      text-transform: capitalize;
-      html[lang="ar"],
-      html[lang="he"] {
-        &:after {
-          content: ' ←';
-        }
-      }
-      &:after {
-        content: ' →';
-      }
```

Comment:
Arrows have never been displayed in the task nav buttons AFAIK, so deleting this theme because there's no documentation about it.